emulation/README.md

```diff
@@ -0,0 +1,19 @@
+# FRC Robot Code Emulation
+
+## Overview
+The emulator is the Synthesis tool designed to help users test their FRC robot code ([Getting started with FRC control system](https://wpilib.screenstepslive.com/s/currentCS "FRC Control System")). Users can upload their own user code as the FRC user programs they would normally deploy to the RoboRIO. Its normal communication with hardware is then redirected to the [engine](../engine "Engine Source") for simulation and testing.
```

Comment:
```suggestion
The emulator is the Synthesis tool designed to help users test their FRC robot code ([Getting started with FRC control system](https://docs.wpilib.org "FRC Control System")). Users can upload their own user code as the FRC user programs they would normally deploy to the RoboRIO. Its normal communication with hardware is then redirected to the [engine](../engine "Engine Source") for simulation and testing.
```

Reply:
Implemented. Thanks for the suggestion. Guess they updated the docs link.